sdk/apis/kubermatic/v1/datacenter.go

```diff
@@ -860,11 +860,31 @@ type DatacenterSpecKubevirt struct {
 
 // ProviderNetwork describes the infra cluster network fabric that is being used.
 type ProviderNetwork struct {
-	Name                 string `json:"name"`
-	VPCs                 []VPC  `json:"vpcs,omitempty"`
-	NetworkPolicyEnabled bool   `json:"networkPolicyEnabled,omitempty"`
+	Name string `json:"name"`
+	VPCs []VPC  `json:"vpcs,omitempty"`
+	// Deprecated: Use dc.ProviderNetwork.NetworkPolicy.Enabled instead.
+	NetworkPolicyEnabled bool          `json:"networkPolicyEnabled,omitempty"`
+	NetworkPolicy        NetworkPolicy `json:"networkPolicy,omitempty"`
```

Comment:
Since this is not a pointer, this field can never be empty, so omitempty does nothing here.